diff --git a/src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java b/src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java
index d63ffdd..32d25c8 100644
--- a/src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java
+++ b/src/main/java/com/google/devtools/build/lib/analysis/config/AutoCpuConverter.java
@@ -55,6 +55,8 @@ public class AutoCpuConverter implements Converter<String> {
               return "ppc";
             case ARM:
               return "arm";
+            case AARCH64:
+              return "aarch64";
             case S390X:
               return "s390x";
             default:
diff --git a/src/main/java/com/google/devtools/build/lib/util/CPU.java b/src/main/java/com/google/devtools/build/lib/util/CPU.java
index e210eb5..313db78 100644
--- a/src/main/java/com/google/devtools/build/lib/util/CPU.java
+++ b/src/main/java/com/google/devtools/build/lib/util/CPU.java
@@ -25,6 +25,7 @@ public enum CPU {
   X86_64("x86_64", ImmutableSet.of("amd64", "x86_64", "x64")),
   PPC("ppc", ImmutableSet.of("ppc", "ppc64", "ppc64le")),
   ARM("arm", ImmutableSet.of("aarch64", "arm", "armv7l")),
+  AARCH64("aarch64", ImmutableSet.of("aarch64")),
   S390X("s390x", ImmutableSet.of("s390x", "s390")),
   UNKNOWN("unknown", ImmutableSet.<String>of());
 
diff --git a/third_party/BUILD b/third_party/BUILD
index 63da36c..b22c63f 100644
--- a/third_party/BUILD
+++ b/third_party/BUILD
@@ -584,6 +584,10 @@ config_setting( 
     name = "arm",
     values = {"host_cpu": "arm"},
 )
+config_setting(
+    name = "aarch64",
+    values = {"host_cpu": "aarch64"},
+)
 
 config_setting(
     name = "freebsd",
